TradeItIosTicketSDK2/TradeItConnector.swift

```diff
@@ -74,6 +74,10 @@ internal extension TradeItConnector {
 
         if result?.isError() == true { // Server sent an ERROR response so try create a TradeItErrorResult
             result = TradeItResultTransformer.transform(targetClassType: TradeItErrorResult.self, json: json)
+        } else if result?.isSecurityQuestion() == true {
+            result = TradeItResultTransformer.transform(targetClassType: TradeItSecurityQuestionResult.self, json: json)
+        } else if result?.isReviewOrder() == true {
+            result = TradeItResultTransformer.transform(targetClassType: TradeItPreviewOrderResult.self, json: json)
```

Comment:
Are there any other possible values for this result status?

Reply:
No the other one is success but is not handle here